benchmarks/benchmarks/bench_linalg.py

```diff
@@ -112,11 +112,14 @@ class Einsum(Benchmark):
     def setup(self, dtype):
         self.a = np.arange(2900, dtype=dtype)
         self.b = np.arange(3000, dtype=dtype)
+        self.b1 = np.arange(240000, dtype=dtype).reshape(400, 600)
         self.c = np.arange(24000, dtype=dtype).reshape(20, 30, 40)
         self.c1 = np.arange(1200, dtype=dtype).reshape(30, 40)
+        self.c2 = np.arange(480000, dtype=dtype)
+        self.c3 = np.arange(600, dtype=dtype)
         self.d = np.arange(10000, dtype=dtype).reshape(10,100,10)
```

Comment:
It would be good at some point to rename these with meaningful names. Is there a reason to have the different size variants? Perhaps a very small variant would be good as well.

Reply:
@mattip I found that meaningless naming is pretty common in every `bench `module, should we open a separate rename PR? I want to focus on the universal intrinsics transform process in this PR.

Reply:
ok. What about adding a very small variant as well?

Reply:
I doubt that small variant is not enough to evaluate the degree of optimization, I will test with small size and provide a report anyway.